Source/IO.cpp

```diff
@@ -394,6 +407,26 @@ PeleC::setPlotVariables()
     }
   }
 
+#ifdef PELEC_USE_SOOT
+  if (plot_soot && add_soot_src) {
+    for (int i = 0; i < NumSootVars; i++) {
+      amrex::Amr::addStatePlotVar(desc_lst[State_Type].name(FirstSootVar + i));
+    }
+  } else {
+    for (int i = 0; i < NumSootVars; i++) {
+      amrex::Amr::deleteStatePlotVar(
+        desc_lst[State_Type].name(FirstSootVar + i));
+    }
+  }
+  int plot_reactions = 1;
```

Comment:
the `plot_reactions` bit should probably be moved outside the `PELEC_USE_SOOT` ifdef condtion?

Reply:
Yeah. I had it in there because I didn't want to mess with the non-soot stuff but it really shouldn't be in there.